lib/matplotlib/backends/backend_wx.py

```diff
@@ -58,6 +59,15 @@ def DEBUG_MSG(string, lvl=3, o=None):
 IDLE_DELAY = 5  # Documented as deprecated as of Matplotlib 3.1.
 
 
+# lru_cache holds a reference to the App and prevents it from being gc'ed.
+@functools.lru_cache()
```

Comment:
```suggestion
@functools.lru_cache(maxsize=1)
```
Not that we save loads of memory, but this makes it a bit more obvious that this is a special usage of `lru_cache`.

Reply:
sure